src/app/pages/single-recipe/single-recipe.component.ts

```diff
@@ -0,0 +1,72 @@
+import { Component, OnInit } from '@angular/core';
+import { RecipesService } from 'src/app/services/recipies/recipes.service';
+
+@Component({
+  selector: 'app-single-recipe',
+  templateUrl: './single-recipe.component.html',
+  styleUrls: ['./single-recipe.component.scss']
+})
+export class SingleRecipeComponent implements OnInit {
+  data?: any;
+
+  constructor(private recipesService: RecipesService) {}
+  recipes: any[] = [];
+  recipe: any;  
+
+  ngOnInit() {
+    this.recipesService.getRecipies().subscribe(
```

Comment:
use a service method to get only one recipe, in order to avoid duplicate recipes content